### DISCUSSION TITLE
File without CIL image

### DISCUSSION THREAD

**Discussion by littlecat-dev** [Pryaxis/TShock#2910] [Q&A]
Latest TShock. When i run server, it doesn't start

```
mono TShock.Server.exe

Cannot open assembly 'TShock.Server.exe': File does not contain a valid CIL image
```

Linux Debian 11, installed mono-complete package.

**Comment by sgkoishi** [comment]
See https://github.com/Pryaxis/TShock/discussions/2843#discussioncomment-4349572 .

**Reply by littlecat-dev** [reply]
https://github.com/Pryaxis/TShock/discussions/2843#discussioncomment-4351214

Huh.. how to run this file from Linux version- TShock.Server

**Reply by sgkoishi** [reply]
> To run the TShock.Server file on a Linux system, follow these steps:
> * Open a terminal window.
> * Navigate to the directory where the TShock.Server file is located using the cd command.
> * Make the TShock.Server file executable by using the chmod command: chmod +x TShock.Server
> * Run the TShock.Server file by typing ./TShock.Server in the terminal.
> * The TShock server should start running. You can connect to the server using a client, such as a web browser or a game client.
>
>Note: Before running the TShock.Server file, make sure that you have all the required dependencies installed on your system. You can refer to the TShock documentation or the developer's website for more information on the required dependencies.

<sup>This answer is generated by [ChatGPT](https://openai.com/blog/chatgpt/) by OpenAI.</sup>
